### PR TITLE
Update validator related screens

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: app/i18n/en.json [hunk 1]
```diff
@@ -184,6 +184,10 @@
       "title": "Redelegation is Locked",
       "message": "You have redelegated to this validator recently, please wait for 21 days after redelegation to unlock."
     },
+    "Status": {
+      "Active": "有效",
+      "Inactive": "無效"
```
Comment (williamchong007) [inline app/i18n/en.json:189]: Should just use the english words

File: app/i18n/zh-Hant-HK.json [hunk 2]
```diff
@@ -186,6 +186,10 @@
       "title": "轉移委託已被鎖上",
       "message": "你最近曾轉移委託至這驗證人，轉移委託後21天將會被解鎖。"
     },
+    "Status": {
+      "Active": "有效",
+      "Inactive": "無效"
```
Comment (williamchong007) [inline app/i18n/zh-Hant-HK.json:191]: 話人 "無效" 會唔會有d奇怪，有種怪責嘅感覺， @nnkken 有無話中文一般點叫？

File: app/components/validator-list/validator-list.tsx [hunk 3]
```diff
@@ -20,12 +20,23 @@ export class ValidatorList extends React.Component<Props> {
...
+  }
 
   render() {
+    const { sortedValidatorList: list } = this.props.chain
     return (
       <View style={this.props.style}>
-        {this.props.chain.sortedValidatorList.filter(this.filterList).map(this.renderItem)}
+        {(
+          this.props.isShort
+            ? list.filter(this.filterList).slice(0, 10)
```
Comment (williamchong007) [inline app/components/validator-list/validator-list.tsx:37]: Should 10 be a configurable number from either code or remote config?

File: app/i18n/zh-Hant-HK.json (hunk 2 above)
Comment (nwingt) [reply app/i18n/zh-Hant-HK.json:191]: I just copied from Big Dipper

Comment (williamchong007) [reply app/i18n/zh-Hant-HK.json:191]: 感覺用 `候選` `後備` 之類比較適合

File: app/components/validator-list/validator-list.tsx (hunk 3 above)
Comment (nwingt) [reply app/components/validator-list/validator-list.tsx]: Now can config through props

File: app/i18n/zh-Hant-HK.json [hunk 4]
```diff
@@ -187,8 +187,8 @@
       "message": "你最近曾轉移委託至這驗證人，轉移委託後21天將會被解鎖。"
     },
     "Status": {
-      "Active": "有效",
-      "Inactive": "無效"
+      "Active": "活躍驗證人",
+      "Inactive": "閒置驗證人"
```
Comment (williamchong007) [inline app/i18n/zh-Hant-HK.json:191]: just 活躍 and 閒置 will do in this case?

**Pull request merged**